client/api/course.js

```diff
@@ -1,41 +1,42 @@
+import path from 'path';
 import request from './request';
 
-const url = {
-  users: (courseId, userId = '') => `/courses/${courseId}/users/${userId}`
+const urls = {
+  base: () => '/courses',
```

Comment:
Why is `base` prop function? Is it more consistent to use it like normal property like in other `api` files?
`base: '/courses'` @vladimyr

Reply:
I was following _imaginary_ rule: if any of ulrs are dynamic, make all others functions too :bowtie: